src/test/scala/ironoxide/v1/FullIntegrationTest.scala

```diff
@@ -58,14 +59,26 @@ class FullIntegrationTest extends AsyncWordSpec with Matchers with EitherValues
       primaryTestUserSigningPrivateKeyBytes
     )
 
-  val sdk = IronOxide.initialize[IO](deviceContext).unsafeRunSync
+  val defaultConfig = IronOxideConfig()
+  val shortConfig = IronOxideConfig(PolicyCachingConfig(), Some(Duration(5, "millis")))
```

Comment:
Is this the recommended Scala way of specifying the units? A quick google search indicated that `java.util.concurrent.TimeUnit.MILLISECONDS` might be an alternative.

Reply:
I didn't see a recommended way, and both ways are referenced equally in the documentation. I originally used the string representation as it was more succinct, but I think I do prefer the `TimeUnit` way for the safety of using set values.
I changed it to `scala.concurrent.duration.MILLISECONDS`